lib/home/profile/report/issue_report.dart

```diff
@@ -120,15 +93,14 @@ class _IssueReportScreenState extends State<IssueReportScreen> {
                   }
                   return null;
                 },
-                onSaved: (value) {
-                  _message = value ?? '';
-                },
+                onSaved: (value) => _message = value ?? '',
               ),
               const SizedBox(height: 16.0),
               ElevatedButton(
                 onPressed: () {
                   if (_formKey.currentState!.validate()) {
                     _formKey.currentState!.save();
+                    backendPrint('$_name $_email $_message');
```

Comment:
The `_name` `_email` `_message` were triggering some [`unused_field`](https://dart.dev/tools/diagnostic-messages#unused_field) warnings, so I just added this line to get rid of those.